editor/modules/aariba/ace.ts

```diff
@@ -118,16 +118,17 @@ class AaribaFoldMode extends FoldMode {
         if (match) {
             let i = match.index;
 
-            if (match[1])
+            if (match[1] && i)
```

Comment:
So you actually mix bugfix and dependencies update d: (yeah, it was a vicious trap)